include/ygm/detail/comm_impl.hpp

```diff
@@ -15,43 +15,61 @@
 #include <ygm/detail/mpi.hpp>
 #include <ygm/detail/ygm_cereal_archive.hpp>
 #include <ygm/meta/functional.hpp>
+#define test_buffer_capacity 50
 
```

Comment:
This size is just for testing. I noticed the buffer capacity is set to some integer value so I replaced that with this preprocessor variable. We can just change this to whatever we want.